components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/js/dialog/export-file-dialog.js

```diff
@@ -29,252 +29,27 @@ define(['require', 'lodash', 'jquery', 'log', 'backbone', 'file_browser', 'boots
-                    var workspaceServiceURL = app.config.services.workspace.endpoint;
-                    var exportServiceURL = workspaceServiceURL + "/export";
-                    var activeTab = app.tabController.activeTab;
-                    var siddhiFileEditor= activeTab.getSiddhiFileEditor();
-                    var config = siddhiFileEditor.getContent();
-                    var payload = "location=" + btoa(options.location) + "&configName=" + btoa(options.configName)
-                        + "&config=" + (btoa(config));
-
-                    $.ajax({
-                        url: exportServiceURL,
-                        type: "POST",
-                        data: payload,
-                        contentType: "text/plain; charset=utf-8",
-                        async: false,
-                        success: function (data, textStatus, xhr) {
-                            if (xhr.status == 200) {
-                                exportConfigModal.modal('hide');
-                                log.debug('file exported successfully');
-                                callback(true);
-                            } else {
-                                exportWizardError.text(data.Error);
-                                exportWizardError.show();
-                                callback(false);
-                            }
-                        },
-                        error: function(res, errorCode, error){
-                            var msg = _.isString(error) ? error : res.statusText;
-                            if(isJsonString(res.responseText)){
-                                var resObj = JSON.parse(res.responseText);
-                                if(_.has(resObj, 'Error')){
-                                    msg = _.get(resObj, 'Error');
-                                }
-                            }
-                            exportWizardError.text(msg);
-                            exportWizardError.show();
-                            callback(false);
-                        }
-                    });
-                }
+                var activeTab = app.tabController.activeTab;
+                var siddhiFileEditor= activeTab.getSiddhiFileEditor();
+                var config = siddhiFileEditor.getContent();
+                var payload = new Blob([config], {type: "text/plain; charset=utf-8"});
+                var downloadLink = document.createElement("a");
+                    downloadLink.download = title;
+                    downloadLink.innerHTML = "export File";
+                    downloadLink.href = window.URL.createObjectURL(payload);
+                    downloadLink.onclick = destroyClickedElement;
+                    downloadLink.style.display = "none";
+                    document.body.appendChild(downloadLink);
+                    window.URL.revokeObjectURL(payload);
+                    downloadLink.click();
+              function destroyClickedElement(event) {
```

Comment:
@erangatl please format the code to indent properly

Reply:
Fixed with #1690